### PR TITLE
Changes to Organ-Stabilizing property

### PR DESCRIPTION
**Pull request by BeagleGaming1**
# About the pull request

Changes Organ-Stabilizing from an on/off flag to being dependent on the organs damage

# Explain why it's good for the game

Makes organ damage matter more if you get injured enough

# Changelog

:cl:
balance: Changes to research Organ-stabilizing property
/:cl:

### CODE REVIEW

**Review by morrowwolf — CHANGES_REQUESTED** [review]
I think having it be a reduction in effect rather than complete turn off would be better. It would be less effective as the organ damage became worse.

Peri should be a way to limp away as someone with organ damage. I'd like a negative effect of some sort associated.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

File: code/modules/organs/organ_internal.dm [hunk 1]
```diff
@@ -8,17 +8,28 @@
 /datum/internal_organ
 	var/name = "organ"
 	var/mob/living/carbon/human/owner = null
-	var/vital //Lose a vital limb, die immediately.
-	var/damage = 0 // amount of damage to the organ
+	// If you lose a vital organ, you die immediately
```
Comment (BeagleGaming1) [inline code/modules/organs/organ_internal.dm:11]:
```suggestion
	/// If you lose a vital organ, you die immediately
```

**Comment by morrowwolf** [comment]
Gonna come back to this in the future

**Pull request closed without merging**